communitybridge/2020/q2/project_ideas.md

```diff
@@ -87,4 +87,10 @@ Project maintainers and mentors, please submit the ideas below (under the Propos
 - Mentor(s): Lucas Servén Marín (@squat), Bartlomiej Plotka (@bwplotka)
 - Issue: https://github.com/thanos-io/thanos/issues/2488
  
+#### CommunityBridge: Per Request Query Tracking and Limiting
```

Comment:
@bwplotka you don't need to mention "CommunityBridge" here, everything else looks good :)

Reply:
Ups, copy pasta, fixed. Thanks for spotting!